consensus/safety-rules/src/persistent_safety_storage.rs

```diff
@@ -47,52 +49,65 @@ impl PersistentSafetyStorage {
     /// Use this to instantiate a PersistentStorage for a new data store, one that has no
     /// SafetyRules values set.
     pub fn initialize(
-        mut internal_store: Storage,
+        mut storage: Storage,
         author: Author,
         consensus_private_key: Ed25519PrivateKey,
         execution_private_key: Ed25519PrivateKey,
         waypoint: Waypoint,
     ) -> Self {
         Self::initialize_(
-            &mut internal_store,
+            &mut storage,
             author,
             consensus_private_key,
             execution_private_key,
             waypoint,
         )
         .expect("Unable to initialize backend storage");
-        Self { internal_store }
+        Self { storage }
     }
 
     fn initialize_(
-        internal_store: &mut Storage,
+        storage: &mut Storage,
         author: Author,
         consensus_private_key: Ed25519PrivateKey,
         execution_private_key: Ed25519PrivateKey,
         waypoint: Waypoint,
     ) -> Result<()> {
-        internal_store.import_private_key(CONSENSUS_KEY, consensus_private_key)?;
-        internal_store.import_private_key(EXECUTION_KEY, execution_private_key)?;
-        internal_store.set(EPOCH, Value::U64(1))?;
-        internal_store.set(LAST_VOTED_ROUND, Value::U64(0))?;
-        internal_store.set(OWNER_ACCOUNT, Value::String(author.to_string()))?;
-        internal_store.set(PREFERRED_ROUND, Value::U64(0))?;
-        internal_store.set(WAYPOINT, Value::String(waypoint.to_string()))?;
-        internal_store.set(
+        storage.import_private_key(CONSENSUS_KEY, consensus_private_key)?;
```

Comment:
the reason for the naming was because it distinguishes it from the fact that this is called PersistentSafetyStore